src/catch2/catch_config.cpp

```diff
@@ -80,6 +81,26 @@ namespace Catch {
             } );
         }
 
+#if defined( CATCH_CONFIG_BAZEL_SUPPORT )
+        // Register a JUnit reporter for Bazel. Bazel sets an enviorment
+        // variable with the path to XML output. If this file is written to
+        // during test, Bazel will not generate a default XML output.
+        // This allows the XML output file to contain higher level of detail
+        // than what is possible otherwise.
+#if defined( CATCH_PLATFORM_WINDOWS )
```

Comment:
This should check for `_MSC_VER` instead, because `CATCH_PLATFORM_WINDOWS` is also defined for MinGW, which uses different syntax for warning suppression pragmas (and also doesn't complain about plain `getenv`)